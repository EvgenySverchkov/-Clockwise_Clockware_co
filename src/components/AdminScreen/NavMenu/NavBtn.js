import React from "react";

export default ({ title, children }) => (
  <div className="dropdown mr-1">
    <button
      className="btn btn-secondary dropdown-toggle"
      href="#"
      id="dropdownMenuLink"
      data-toggle="dropdown"
      aria-haspopup="true"
      aria-expanded="false"
    >
      {title}
    </button>
    <div className="dropdown-menu" aria-labelledby="dropdownMenuLink">
      {children}
    </div>
  </div>
);
