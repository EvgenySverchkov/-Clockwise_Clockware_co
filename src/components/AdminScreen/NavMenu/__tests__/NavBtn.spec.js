import React from "react";
import {create} from "react-test-renderer";

import NavBtn from "../NavBtn";

it("Test of <NavBtn /> on admin side", ()=>{
    const mockProps = {
        title: "Title"
    }
    const component = create(
        <NavBtn {...mockProps}>
            <button>Some text1</button>
            <button>Some text2</button>
        </NavBtn>);
    expect(component.toJSON()).toMatchSnapshot();
});