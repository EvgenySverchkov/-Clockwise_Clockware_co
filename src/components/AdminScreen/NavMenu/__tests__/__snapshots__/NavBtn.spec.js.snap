// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test of <NavBtn /> on admin side 1`] = `
<div
  className="dropdown mr-1"
>
  <button
    aria-expanded="false"
    aria-haspopup="true"
    className="btn btn-secondary dropdown-toggle"
    data-toggle="dropdown"
    href="#"
    id="dropdownMenuLink"
  >
    Title
  </button>
  <div
    aria-labelledby="dropdownMenuLink"
    className="dropdown-menu"
  >
    <button>
      Some text1
    </button>
    <button>
      Some text2
    </button>
  </div>
</div>
`;
